src/ocm/components/clusterConfiguration/review/ReviewStep.tsx

```diff
@@ -12,15 +19,29 @@ import { useStateSafely } from '../../../../common/hooks';
 import { selectCurrentClusterPermissionsState } from '../../../selectors';
 import ReviewPreflightChecks from './ReviewPreflightChecks';
 import ReviewSummary from './ReviewSummary';
+import { getLimitedFeatureSupportLevels } from '../../../../common/components/featureSupportLevels/utils';
+import { useTranslation } from '../../../../common/hooks/use-translation-wrapper';
+
 import './ReviewCluster.css';
 
 const ReviewStep = ({ cluster }: { cluster: Cluster }) => {
   const { addAlert } = useAlerts();
   const clusterWizardContext = useClusterWizardContext();
   const { isViewerMode } = useSelector(selectCurrentClusterPermissionsState);
   const [isStartingInstallation, setIsStartingInstallation] = useStateSafely(false);
+  const featureSupportLevelData = useFeatureSupportLevel();
+  const { t } = useTranslation();
   const dispatch = useDispatch();
 
+  const hasSupportLevel = React.useMemo<boolean>(() => {
+    try {
+      getLimitedFeatureSupportLevels(cluster, featureSupportLevelData, t);
+      return true;
+    } catch (e) {
+      return false;
+    }
```

Comment:
Why do we need 2 return statements?

```suggestion
    let theAnswer = false;
    try {
      theAnswer = getLimitedFeatureSupportLevels(cluster, featureSupportLevelData, t);
    } catch {
      // pass...
    }
    
    return theAnswer;
```

Reply:
Fixed it